homeassistant/components/climate/honeywell.py

```diff
@@ -132,9 +133,9 @@ def target_temperature(self):
             return None
         return self._target_temperature
 
-    def set_temperature(self, temperature):
+    def set_temperature(self, **data):
```

Comment:
OHHHH - sorry, the issue is that the unit test is using poisitonal args - need to change to keywords